yrs/src/block.rs

```diff
@@ -39,6 +50,107 @@ pub enum Block {
     GC(GC),
 }
 
+impl Block {
+    pub fn as_item(&self) -> Option<&Item> {
+        match self {
+            Block::Item(item) => Some(item),
+            _ => None
+        }
+    }
+
+    pub fn as_item_mut(&mut self) -> Option<&mut Item> {
+        match self {
+            Block::Item(item) => Some(item),
+            _ => None
+        }
+    }
+
+    pub fn is_deleted(&self) -> bool {
+        match self {
+            Block::Item(item) => item.deleted,
+            Block::Skip(_) => false,
+            Block::GC(_) => false,
```

Comment:
`GC` is a garbage-collected deleted operation. So this should return true.

Reply:
👍  I did that but apparently erased while cleaning up commit.